src/builder.rs

```diff
@@ -99,6 +99,107 @@ impl TransactionBuilder {
     }
 }
 
+/// Builds a ReissueRequest from a ReissueTransaction and
+/// any number of (input) DBC hashes with associated ownership share(s).
+#[derive(Default)]
+pub struct ReissueRequestBuilder {
+    pub reissue_transaction: Option<ReissueTransaction>,
```

Comment:
Lets make this field not Optional. Our constructor already expects a reissue transaction and it'll save us from an additional error to check